README.md

```diff
@@ -270,6 +272,9 @@ Creates square region for joystick movement
 ### `options.dynamicPage` defaults to false
 Enable if the page has dynamically visible elements such as for Vue, React, Angular or simply some CSS hiding or showing some DOM.
 
+### `options.follow` defaults to false
+Enable if the joystick must follow the thumbstick when it reaches the border.
```

Comment:
That sounds weird, I know you wanted to follow the previous sentence, but I don't think it's working great in this context.

Maybe something like:

> Makes the joystick follow the thumbstick when it reaches the border.

WDYT?